rtloader/three/three.cpp

```diff
@@ -129,8 +132,10 @@ py_info_t *Three::getPyInfo()
         setError("could not allocate a py_info_t struct");
         return NULL;
     }
-
-    info->version = Py_GetVersion();
+    const char* v = Py_GetVersion();
+    if(v){
+        info->version = strdup(v);
+    }
```

Comment:
If `v` is null `info->version` is not initialized. `v` is probably never null and so removing the `if` is also an option (`info->version = strdup(Py_GetVersion())`)